app/models/person.rb

```diff
@@ -1,17 +1,16 @@
 class Person < ActiveRecord::Base
-  #code to get next un-used id # in the 800 range
-  #((801...900).to_a - (array_from_db_of_taken_numbers))[0]
+  include ActiveModel::ForbiddenAttributesProtection
   before_save :title_order
 
-  attr_accessible :firstname, :lastname, :status, :icsid, :department, :city, :state, :zipcode, :start_date, :end_date , :title, :gender, :date_of_birth,:division1, :division2, :channels_attributes, :title_ids, :title_order, :comments
-
-  #Having a condition on this association allows all the chaining magic to happen.
-  #Could I use a named scope, and/or could I have another association for 'active_certs' ?
   has_many :certs, :conditions => {:status =>'Active' }
-
-  # Since phones and emails derive off of channels, is "has_many :channels" below redundant?
```

Comment:
has_many channels is necessary if you ever want to check all of a `person`'s channels, without thinking about whether they are `phone`s or `email`s or whatnot. `person.channels` will give you all of the means of contacting a person. I'll let you decide if you think that's valuable or not.